regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -52,28 +52,42 @@ <h3 class="toc-type">Table of Contents</h3>
     <section id="comment-review" data-page-type="comment-review" data-doc-id="{{doc_number}}">
       <h2>Review</h2>
       <div id="comment"></div>
+    </section>
+  </div>
+
+  <script id="comment-template" type="text/template">
+    <% if (comments.length) { %>
+      <div>
+        <% _.each(comments, function(comment, index) { %>
+          <div>
+            <h3><%= comment.label %></h3>
+            <div><%= comment.comment %></div>
+            <% if (comment.files.length) { %>
+              <div>
+                Attachments included
+                <ul>
+                  <% _.each(comment.files, function(file) { %>
+                    <li><%= file.name %> Preview</li>
+                  <% }) %>
+                </ul>
+              </div>
+            <% } %>
+          </div>
+          <div>Edit comment</div>
+          <% if (index < comments.length - 1) { %>
+            <div>-----</div>
```

Comment:
Long term, I'd think this would be a css border property or at the very least, an `hr`, yes?